charmcraft/commands/store/__init__.py

```diff
@@ -52,56 +52,61 @@ def get_name_from_metadata():
 
 
 class LoginCommand(BaseCommand):
-    """Log into the store."""
+    """Login to Charmhub."""
```

Comment:
AIUI the verb is "log in", and "login" the noun, so this (and a few other below) should be
```suggestion
    """Log in to Charmhub."""
```
(not "into" as the emphasis is on the "in")

Reply:
Srsly? :)